### ISSUE TITLE
Add `multiline` value for `forbid:` option for both `braces` & `brackets` rules

### ISSUE THREAD

**Issue by rgoldberg** [adrienverge/yamllint#752]
Could you add a `multiline` value for the `forbid:` option for both the `braces` & `brackets` rules?

If used:

```yaml
# PASS
a: []
b: [a, b, c]
c:
  - a
  - b
  - c
d: {}
e: {a: 1, b: 2}
f:
  a: 1
  b: 2

# FAIL
g: [
  ]
h: [
  a,
  b,
  c
]
I: {
  }
j: {
  a: 1,
  b: 2
}
```
